src/IntrusivePtr.h

```diff
@@ -127,6 +127,10 @@ struct IntrusivePtr {
         return ptr == other.ptr;
     }
 
+    bool operator < (const IntrusivePtr &other) const {
```

Comment:
Isn't style for operator overloading in Halide to remove spaces, ie

    bool operator<(...)